jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -8259,12 +8277,12 @@ fi
       ;;
   esac
 
-  if test "x$with_noncompressedrefs" = x; then
-    OPENJ9_BUILDSPEC="${OPENJDK_BUILD_OS}_${OPENJ9_CPU}_cmprssptrs"
-    OPENJ9_LIBS_SUBDIR=compressedrefs
-  else
+  if test "${with_noncompressedrefs+set}" = set || test "x$OPENJDK_TARGET_CPU_BITS" = x32; then
```

Comment:
This (generated-configure.sh) is a generated file.  You need to modify the master file (i.e. https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/jdk/make/closed/autoconf/custom-hook.m4) and then run common/autoconf/autogen.sh to regenerate. See Appendix A in https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/README-builds.html

Reply:
Already generated with custom-hook.m4.